core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.scss

```diff
@@ -420,9 +426,8 @@ transform: rotateY(-50%)
   width: calc((100% - 2rem)/2);
   background-color: $color-white;
   box-shadow: $env-box-shadow;
+  border-radius: 10px 10px 0px 0px;
```

Comment:
make border radius on every corner and not just on top corners

Reply:
Addressed